Bundle/FormBundle/Resources/public/js/link.js

```diff
@@ -0,0 +1,24 @@
+/**
```

Comment:
This script is still used there:
https://github.com/FriendsOfVictoire/WidgetFAQBundle/blob/master/Resources/views/form.html.twig#L56
https://github.com/FriendsOfVictoire/WidgetAlertButtonBundle/blob/master/Resources/views/new.html.twig#L10
https://github.com/FriendsOfVictoire/WidgetMenuBundle/blob/master/Resources/public/edit/js/menu.js#L123
https://github.com/FriendsOfVictoire/WidgetMenuBundle/blob/master/Resources/public/edit/js/menu.js#L128

These widgets needs refactor if we want to delete this script